src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -525,6 +525,23 @@ paths:
           $ref: '#/components/responses/AedSearchResults'
       operationId: search-anonymous-elector-document-summaries
       parameters:
+        - name: page
+          description: The page number being requested (with the first page being 1). Defaulted to 1 if not supplied
+          schema:
+            type: integer
+            minimum: 1
+          example: 1
+          in: query
+          required: true
+        - name: pageSize
+          description: The number of results to return in each page. Defaulted to 100 if not supplied
```

Comment:
Is this on the assumption the UI is going to be ready?

Reply:
Even if UI doesn't supply page or pageSize, the controller defaults them to 1 and 100, so shouldn't break UI.

Reply:
![Uploading image.png…]()

Reply:
<img width="849" alt="image" src="https://user-images.githubusercontent.com/73763010/236178311-834930b9-8d54-47b3-be72-deeb9fffdd35.png">

Reply:
What if there are > 100 results? I thought we were going to set it to something high, like 1000 to start wtih?

Reply:
Yes sure, Could change that in next subtasks.

Reply:
As per great @matt-wills-valtech there are only around 3570 anonymous electors, so chances of AEDs among some ero's applications going above are going to be less than 100, except one of two EROS which has more AEDs. 😉

Reply:
Perhaps the best thing is to make a decision on this tomorrow, depending on how the API & UI dev goes, so I'll approve this 👍